app/3.0/docs/glossary.md

```diff
@@ -4,14 +4,11 @@ title: "Polymer Glossary"
 
 <!-- toc -->
 
-<div>
-{% include 'outdated.html' %}
-</div>
-
 ## A
 
-### Analyzer
-See [Polymer Analyzer](#polymer-analyzer).
+### AMD modules
+
+AMD (Asynchronous Module Definition) modules use the `define()` function to write modular JavaScript code that can be loaded asynchronously. Polymer CLI can convert [ES modules](#es-modules) to AMD modules so that older browsers can load them.
```

Comment:
I think the most relevant information here is that AMD modules are loaded using a (small) library, so they can be used with browsers that don't support native ES modules.